### DISCUSSION TITLE
There is a way to customize the test title?

### DISCUSSION THREAD

**Discussion by nelson6e65** [pestphp/pest#485] [Q&A] [answered]
For example, 

```php
it('paginates results for', function (string $resource, bool $authenticate, int $statusCode) {
    // ...
})
    ->with('resources')
    ->with('authenticationAware')
;
```

I get this message:

```
✓ it paginates results for with ('appointments') / (true, 200)
```

But I want to get rid of 'with' suffix:

```diff
-✓ it paginates results for with ('appointments') / (true, 200)
+✓ it paginates results for ('appointments') / (true, 200)
```

**There is a way to achieve this?**

**Comment by fabio-ivona** [comment, marked as answer]
sorry, It is not possible with the actual implementation

**Reply by owenvoke** [reply]
Yeah, I'd just do `paginates results` so you end up with `it paginates results with ...`

**Reply by nelson6e65** [reply]
Sad.

Yes, I had like that @owenvoke, but this was just an example; my company wanted to use messages in Spanish instead, so my question was more generic. Thanks, anyway.
